api/db/migrations/20251009145131_create-tutorials-table.js

```diff
@@ -0,0 +1,40 @@
+const TABLE_NAME = 'tutorials';
+const RELATIONSHIPS_TABLE_NAME = 'tutorials-tutorial_tags';
```

Comment:
```suggestion
const RELATIONSHIPS_TABLE_NAME = 'tutorials-tutorial_tags_link';
```

J'aurais mis ça comme ça pour être raccord avec les tables de static-courses et tags

Reply:
La convention habituelle pour une table de jointure est de nommer la table avec les noms des 2 tables jointes séparées par un tiret.
L'usage du mot `link` est redondant.
On a préféré suivre la convention habituelle et on pourra renommer les tables qui ne la suivent pas aujourd'hui.